inputfiles/overridingTypes.jsonc

```
@@ -2349,6 +2349,20 @@
                 "constructor": {
                     "signature": {
                         "0": {
+                            "param": [
+                                {
+                                    "name": "underlyingSource",
+                                    "optional": false,
+                                    "overrideType": "UnderlyingByteSource"
```

```suggestion
                                    "type": "UnderlyingByteSource"
```

(I believe this works)